.github/workflows/ci.yaml

```diff
@@ -46,7 +46,8 @@ jobs:
         run: |
           tools_dir=cwl
           new_version=${{ env.SHORT_SHA }}
-          pipenv run utils/bump_cwl_version.py $tools_dir $new_version
+          templates_dir=template
+          pipenv run utils/bump_cwl_version.py $tools_dir $new_version $templates_dir
```

Comment:
to make it more generic how about passing in a json `input` or `data` dictionary here.  something like..
```
{
  "data": {
    "version": "1.0.0"
  }
}
```

then you can reference it with `{{ data.version }}` in your mustache templates.  that way you can pass in more than one piece of data.